android/build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
     def isPlay = !taskRequests.contains("foss")
 
     ext {
+        applicationId = "chat.rocket.reactnative"
```

Comment:
On the white label docs, we ask them to set this app id (and other values) on the `gradle.properties`: https://docs.rocket.chat/guides/developer/mobile-apps/whitelabeling-mobile-apps#general-setup-1
This is different on the `single-server` vs `develop` branch, but I agree we can share the same behaviour.

IMO, it's easier if we ask the reader to change everything in one place, if possible.
Can we move this to `gradle.properties` as well?

Reply:
Sure - I'll push the change up